src/sentry/static/sentry/app/views/settings/components/scrollToTop.jsx

```diff
@@ -0,0 +1,20 @@
+import {withRouter} from 'react-router';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class ScrollToTop extends React.Component {
+  static propTypes = {
+    location: PropTypes.object,
+  };
+
+  componentDidUpdate(prevProps) {
+    if (this.props.location !== prevProps.location) {
```

Comment:
will this trigger on an anchor being added to the url?

Reply:
i don't actually know when this could happen, it seems to work well locally for me

Reply:
It's possible, but we don't really have anchors (besides for search which seems to work)